### PR TITLE
 Fix terminal abort request

### PR DESCRIPTION
**Pull request by martinsrenato**
PW-1384 - Terminal API AbortRequest results in exception

### CODE REVIEW

**Comment by rkewlani** [comment]
Can you please check why build is failing ?

File: src/main/java/com/adyen/service/TerminalLocalAPI.java [hunk 1]
```diff
@@ -65,6 +65,10 @@ public TerminalAPIResponse request(TerminalAPIRequest terminalAPIRequest, Securi
 
         String jsonResponse = localRequest.request(jsonEncryptedRequest);
 
+        if (jsonResponse == null || jsonResponse.isEmpty()) {
```
Comment (rkewlani) [inline src/main/java/com/adyen/service/TerminalLocalAPI.java]: do we need to check "ok" here ?

Comment (AlexandrosMor) [reply src/main/java/com/adyen/service/TerminalLocalAPI.java]: "Ok" is not a SaleToPoiResponse. I believe the response should be null

Comment (martinsrenato) [reply src/main/java/com/adyen/service/TerminalLocalAPI.java]: Like @AlexandrosMor , for the local API in this case we only get null or a SaleToPoiResponse object as response, so I don't think there's a need to put "ok" here

**Pull request merged**